amun/api_v1.py

```diff
@@ -108,17 +119,23 @@ def _adjust_default_requests(dict_: dict) -> None:
     dict_['requests'] = new_requests
 
 
+@versionchanged(
+    version="0.6.0",
+    reason="The function now submits an Argo Workflow."
+)
```

Comment:
Why is this needed?

Reply:
Ah, right. I can remove these already for the updated functions.